notebooks/228-clip-zero-shot-image-classification/228-clip-zero-shot-image-classification.ipynb

```diff
@@ -0,0 +1,281 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Zero-shot Image Classification with OpenAI Clip and OpenVINO™\n",
+    "\n",
+    "Zero-shot image classification is a computer vision task to classify images into one of several classes, without any prior training or knowledge of the classes. \n",
+    "\n",
+    "![zero-shot-pipeline](https://user-images.githubusercontent.com/29454499/207773481-d77cacf8-6cdc-4765-a31b-a1669476d620.png)\n",
```

Comment:
Is this image our image, or is this coming from another resource?

Reply:
If we are using this from another source, we must reference it.

Reply:
added

Reply:
Did you add the references (links) to the images?